user/how-to-guides/how-to-open-urls-in-other-qubes.md

```diff
@@ -1,174 +1,105 @@
 ---
 lang: en
 layout: doc
-permalink: /doc/how-to-open-urls-in-other-qubes/
-ref: TBD
-title: How to open URLs/files in other qubes
+permalink: /doc/how-to-open-urls-and-files-in-other-qubes/
+title: How to open URLs and files in other qubes
 ---
 
-*This page is about opening untrusted files and URLs from "secure" offline or
-firewalled qubes in "general-purpose" qubes - or any qubes you see fit. The
```

Comment:
I don't think this is really limited to the "secure/offline -> general/online" use case, so I generalized it.